src/components/PaymentOverview/__snapshots__/test.spec.js.snap

```diff
@@ -75,8 +75,12 @@ exports[`on valid redirect renders an error message on failure 1`] = `
         </div>
       </div>
     </div>
-    <a>
-      some link
+    <a
+      className="utrecht-link utrecht-link--openforms"
+      href="/"
+      onClick={[Function]}
```

Comment:
I'll try not to try to understand snapshots. I'm just happy this gets recorded, I would not have been able to come up with this assertion.

Reply:
yeah I check the output, verified that it's correct and the snapshot just needs updating, which is done via `jest` flags/interactive session, so neither do I.